File: apps/dotnet/Simex/Simex/Models/Cost.cs
using System;
using System.Collections.Generic;

namespace Simex.Models;

public partial class Cost
{
    public int Id { get; set; }

    public int OperationId { get; set; }

    public int CostTypeId { get; set; }

    public int CurrencyTypeId { get; set; }

    public int SenedTypeId { get; set; }

    public decimal Cost1 { get; set; }

    public int CostAmount { get; set; }

    public decimal Sale { get; set; }

    public int SaleAmount { get; set; }

    public virtual CostType CostType { get; set; } = null!;

    public virtual CurrencyType CurrencyType { get; set; } = null!;

    public virtual Operation Operation { get; set; } = null!;

    public virtual SendType SenedType { get; set; } = null!;
}

File: apps/web/frontend/src/interfaces/costs/cost.ts
import type { SendType } from '@interfaces/costs/sendType';
import type { CurrencyType } from '@interfaces/costs/currencyType';
import type { CostType } from '@interfaces/costs/costType';

export interface Cost {
  id: number;
  costType: CostType;
  currencyType: CurrencyType;
  sendType: SendType;
  cost: number;
  costAmount: number;
  sale: number;
  saleAmount: number;
}
